lib/Raven/Stacktrace.php

```diff
@@ -171,7 +171,7 @@ public function addFrame($file, $line, array $backtraceFrame)
      *
      * @throws \OutOfBoundsException If the index is out of range
      */
-    public function removeFrame($index)
+    protected function removeFrame($index)
```

Comment:
This should not be protected, an user may want to remove a given frame from the stacktrace before sending it to the Sentry server